_includes/releases/v22.1/v22.1.12.md

```diff
@@ -0,0 +1,104 @@
+## v22.1.12
+
+Release Date: December 12, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-12-general-changes">General changes</h3>
+
+- Bulk operations now log sanitized connection destinations, for example:
+
+	 `backup planning to connect to destination gs://test/backupadhoc?AUTH=specified&CREDENTIALS=redacted` [#92207][#92207]
+
+<h3 id="v22-1-12-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- Kafka sinks can now (optionally) be configured with a "Compression" field to the `kafka_sink_config` option. This field can be set to "none" (default), "GZIP", "SNAPPY", "LZ4", or "ZSTD". Setting this field will determine the compression protocol used when emitting events. [#91276][#91276]
+
+<h3 id="v22-1-12-operational-changes">Operational changes</h3>
+
+- Logs produced by setting an increased vmodule setting for s3_storage are now directed to the DEV channel rather than STDOUT. [#91960][#91960]
+
+- Introduced a metric (`replicas.leaders_invalid_lease`) that indicates how many replicas are raft group leaders but holding invalid leases. [#91194][#91194]
+
+<h3 id="v22-1-12-db-console-changes">DB Console changes</h3>
+
+- Changed the height of column selector, so it can hint there are more options to be selected once scrolled. [#91910][#91910]
+- Added fingerprint ID in hex format to the Statement Details page and Transaction Details page. [#91959][#91959]
+- Updated tooltip on `SQL Statement Errors` chart on Metrics page. [#92711][#92711]
+
+<h3 id="v22-1-12-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug in which panics triggered by certain DDL statements were not properly recovered, leading to the cluster node crashing. [#91555][#91555]
+- Fixed a panic that could occur when calling st_distancespheroid or st_distancesphere with a spatial object containing an NaN coordinate. This now produces an error, "input is out of range". [#91634][#91634]
+- Fixed a bug that resulted in some retriable errors not being retried during IMPORT. [#90432][#90432]
+- Fixed a bug in `Concat` projection operators for arrays that could cause non-null values to be added to the array when one of the arguments was null. [#91653][#91653]
+- Fixed a bug described in #91069. [#91089][#91089]
```

Comment:
I think you could just use the description in the PR here in 91089:

Previously, `SET DEFAULT NULL` resulted in a column whose `DefaultExpr` is `NULL`. This is problematic when used with `ALTER COLUMN TYPE` where a temporary computed column will be created, hence violating validation that "a computed column cannot have default expression". This is now fixed by setting `DefaultExpr` to `nil` when `SET DEFAULT NULL`.